e2e-tests/src/utilities/gql.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * GraphQL Query and Mutation strings.
+ */
+const gql = {
+  CREATE_MISSION_MODEL: `#graphql
+    mutation CreateMissionModel($model: mission_model_insert_input!) {
+      insert_mission_model_one(object: $model) {
+        id
+      }
+    }
+  `,
+
+  DELETE_MISSION_MODEL: `#graphql
+    mutation DeleteModel($id: Int!) {
+      delete_mission_model_by_pk(id: $id) {
+        id
+      }
+    }
+  `,
+};
+
+export default gql;
```

Comment:
This is awesome. In case anyone else is wondering this `const gql` can eventually be populated with the contents of https://github.com/NASA-AMMOS/aerie-ui/blob/develop/src/utilities/gql.ts, almost verbatim. :)